specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -1666,7 +1666,12 @@
             }
           }
         },
-        "x-ms-long-running-operation": true
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create a VM Extension with suppressFailures enabled": {
+            "$ref": "./examples/CreateAVMWithExtensionSuppressFailureEnabled.json"
```

Comment:
update to "./examples/compute/CreateAVMWithExtensionSuppressFailureEnabled.json"

Reply:
there isn't a compute dir under examples.